code/mod_table/mod_table.cpp

```diff
@@ -297,6 +298,15 @@ void parse_mod_table(const char *filename)
 			}
 		}
 
+		if (optional_string("$Show-subtitle uses pixels:")) {
+			stuff_boolean(&Show_subtitle_uses_pixels);
+			if (Show_subtitle_uses_pixels) {
+				mprintf(("Game Settings Table: Show-subtitle uses pixels\n"));
+			} else {
+				mprintf(("Game Settings Table: Show-subtitle uses percentages\n"));
```

Comment:
Since this is the default value, should this message be printed out even if the `$Show-subtitle uses pixels:` option is not present?

Reply:
I don't think so.  This follows the pattern of other options: they are only logged to the log file if they are explicitly specified somewhere.